### ISSUE TITLE
Drupal WET Distro Accessibility App

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-drupal#296]
Drupal has pretty good accessibility built into it, however it's not perfect.  There's lots that can & should be done to to ensure that all web pages meet WCAG 2.0 AA compliance.  

We're a long ways in Canada from what the BBC has already accomplished http://www.bbc.co.uk/accessibility/best_practice/
1. [Quail API](https://drupal.org/project/quail_api) https://github.com/kevee/quail 
2. [Accessible Helper Module](http://drupal.org/project/accessible)
3. [Readability Analyzer](http://drupal.org/project/readability)
4. [Block ARIA Landmark Roles](http://drupal.org/project/block_aria_landmark_roles)
5. [Page Style](http://drupal.org/project/pagestyle)
6. [HTML Tidy](http://drupal.org/project/htmltidy)

This should all be wrapped up in an app so that it is all configured with appropriate libraries & is tested with the WET framework. 

There is a list of [contributed modules to help with accessibility](http://drupal.org/node/394252) but I've pulled out the best of the lot for WET.  

There are also simple approaches like adding a block which adds [Holmes CSS](https://github.com/redroot/holmes) & simple links to check pages from [WAVE](http://wave.webaim.org/) & [FAE](https://fae.cita.illinois.edu/).

**Comment by mgifford** [comment]
There's also room to improve error handling now that it looks like http://drupal.org/node/742344 might get into D7.

For WCAG 2.0 AA we gotta be looking at examples like:
  https://github.com/wet-boew/wet-boew/wiki/Form-validation

or from the UK, the error page here is excellent:
  http://www.actionforblindpeople.org.uk/other-pages/contact-us/

Might well be a new module to help enhance the error message pages.

**Comment by mgifford** [comment]
Language of parts & tinymce - well, this example is with ckeditor, but...
http://drupal.org/node/1165466

**Comment by sylus** [comment]
Think this discussion is best placed in the accessibility module at drupal.org/project/accessibility which we plan to leverage.

**Comment by mgifford** [comment]
https://drupal.org/project/accessibility isn't going to cover most of the items discussed in this thread.  It's essentially a javascript validator.  It's excellent to leverage that and I look forward to seeing that module as part of WETkit.  Think this will be part of the next RC? 

Language of Parts, Form Validation, & Readability aren't something that can be covered there.
